LICENSE

```diff
@@ -186,7 +186,7 @@ APPENDIX: How to apply the Apache License to your work.
    same "printed page" as the copyright notice for easier
    identification within third-party archives.
 
-Copyright 2012, Chef Software, Inc.
+Copyright [yyyy] [name of copyright owner]
```

Comment:
Don't we fill in this bit??

Reply:
The author of the commit doesn't need to fill in our copyright. They fill in their copyright.